modules/claims_api/app/mailers/claims_api/unsuccessful_report_mailer.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module ClaimsApi
+  class UnsuccessfulReportMailer < ApplicationMailer
+    RECIPIENTS = %w[
+      michael.bastos@oddball.io
+      ryan.link@oddball.io
+      christopher.stone@libertyits.com
+      valerie.hase@va.gov
+      mark.greenburg@adhocteam.us
+      premal.shah@va.gov
+      lee.deboom@oddball.io
+      dan.hinze@adhocteam.us
```

Comment:
nit: is there any sort of gmail groups we can use/create that would allow us to add/remove people from these emails without a code change?

Reply:
We created this methodology originally just to ensure that the emails were sent to the right people but deserves a ticket to figure out how to do something similar to google groups within the VA for certain teams and organizations instead etc.